src/commands/game_commands/music.ts

```diff
@@ -0,0 +1,182 @@
+ * @param textChannelName - The name of the text channel to send the message to
+ * @param voiceChannelName - The name of the voice channel to join
+ * @param message - The original message that triggered the command
+ * @param participants - The list of participants
+ */
+export async function sendBeginGameMessage(
+    textChannelName: string,
+    voiceChannelName: string,
+    message: GuildTextableMessage
+): Promise<void> {
+    const startTitle = state.localizer.translate(
+        message.guildID,
+        "command.play.gameStarting",
+        {
+            textChannelName,
+            voiceChannelName,
+        }
+    );
+
+    const gameInfoMessage: GameInfoMessage = chooseWeightedRandom(
+        await dbContext.kmq("game_messages")
+    );
+
+    const fields: Eris.EmbedField[] = [];
+    if (gameInfoMessage) {
+        fields.push({
+            name: state.localizer.translate(
+                message.guildID,
+                gameInfoMessage.title
+            ),
+            value: state.localizer.translate(
+                message.guildID,
+                gameInfoMessage.message
+            ),
+            inline: false,
+        });
+    }
+
+    await sendInfoMessage(MessageContext.fromMessage(message), {
+        title: startTitle,
+        color: EMBED_SUCCESS_BONUS_COLOR,
+        thumbnailUrl: KmqImages.HAPPY,
+        fields,
+    });
+}
+
+export default class MusicCommand implements BaseCommand {
+    validations = {
+        minArgCount: 0,
+        maxArgCount: 0,
+        arguments: [],
+    };
+
+    aliases = ["radio", "listen"];
+
+    help = (guildID: string): Help => ({
+        name: "music",
+        description: state.localizer.translate(
+            guildID,
+            "command.music.help.description"
+        ),
+        usage: "music",
+        // priority: 1050,
+        examples: [
+            {
+                example: "`,music`",
+                explanation: state.localizer.translate(
+                    guildID,
+                    "command.music.help.example"
+                ),
+            },
+        ],
+    });
+
+    call = async ({ message, channel }: CommandArgs): Promise<void> => {
+        const messageContext = MessageContext.fromMessage(message);
+        const guildID = message.guildID;
+        const session = Session.getSession(guildID);
+        if (session instanceof GameSession) {
+            sendErrorMessage(messageContext, {
+                title: "command.music.failure.existingGameSession.title",
+                description: "command.music.failure.existingGameSession.title",
+            });
+            return;
+        }
+
+        const guildPreference = await getGuildPreference(message.guildID);
+        const textChannel = channel;
+        const gameOwner = KmqMember.fromUser(message.author);
+        const voiceChannel = getUserVoiceChannel(messageContext);
+        const timeUntilRestart = await getTimeUntilRestart();
```

Comment:
Is this copied from play.ts? If so, can we move it somewhere common

Reply:
It is, I'll move it to a precheck